code/datums/outfits/outfit.dm

```diff
@@ -54,6 +57,39 @@
 	else
 		H.equip_to_slot_or_del(I, slot)
 
+/datum/outfit/proc/equip_belt(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(!H)
+		return
+
+	var/obj/item/weapon/storage/belt/B = H.get_equipped_item(slot_belt)
+	if(B)
+		for(var/v in belt_contents)
+			var/number = belt_contents[v]
+			for(var/i in 1 to number)
+				new v(B)
+
+/datum/outfit/proc/equip_accessory(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(!H)
+		return
+
+	var/obj/item/clothing/under/U = H.get_equipped_item(slot_w_uniform)
+	if(U)
+		var/obj/item/clothing/accessory/A = new accessory
+		U.attach_accessory(H, A)
+		if(istype(A, /obj/item/clothing/accessory/storage))
+			var/obj/item/clothing/accessory/storage/S = A
+			for(var/v in accessory_contents)
+				var/number = accessory_contents[v]
+				for(var/i in 1 to number)
+					var/obj/item/I = new v
+					S.hold.insert_into_storage(I)
+		else if(istype(A, /obj/item/clothing/accessory/holster))
+			var/obj/item/clothing/accessory/holster/holster = A
+			var/w_type = accessory_contents[1]
```

Comment:
This assumes that there is an `accessory_contents` list with item length >= 1. This may not be the case.